docs/designers-developers/developers/block-api/block-registration.md

```diff
@@ -195,15 +195,17 @@ example: {
     attributes: {
         cover: 'https://example.com/image.jpg',
     },
-    innerBlocks: {
+    innerBlocks: [
+    {
         name: 'core/paragraph',
         attributes: {
             /* translators: example text. */
             content: __(
                 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent et eros eu felis.'
             ),
-        },
+        }
     },
```

Comment:
This object (lines 199-207) should be indented by four spaces.

Reply:
Thank you @talldan. 

I just have a question.  According to the `.editorconfig` of Gutenberg, it says indent should be tab.
[https://github.com/naogify/gutenberg/blob/docs/update-block-registration/.editorconfig#L14](https://github.com/naogify/gutenberg/blob/docs/update-block-registration/.editorconfig#L14)

The codingstandards of of documentaiotn is different from the one?

Reply:
@naogify That's true, the coding standards say tabs - https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/#spacing.

In my comment above I mostly opted for consistency with the surrounding code which seemed to be indented using four spaces, but checking the rest of the file it seems inconsistent, sometimes tabs, sometimes spaces.

I think it probably doesn't matter so much in documentation, as long as it looks correct. The trouble is that it's harder to enforce without code linting.

Reply:
@talldan We have documentation linting and prettier for docs, not enforced yet as the config and definitions needs some refining, but the basics are in place.

Editor config for setting up Prettier:
https://developer.wordpress.org/block-editor/contributors/document/#editor-config

And the linting uses markdownlint that also can be setup in editor or run using: `npm run lint:md-docs`

Reply:
@mkaz Thanks for the info! I managed to get that working in my editor. Looking forward to being able to write everything on one line, and then magically have it formatted when saving 😄 .

Reply:
@mkaz @talldan 
Thank you for the information,and reply. I understand and will try the markdown linting :)